src/content/docs/fr/tutorial/1-setup/4.mdx

```diff
@@ -0,0 +1,70 @@
+---
+type: tutorial
+title: Stockez votre dépôt en ligne
+description: |-
+  Tutoriel : Construisez votre premier blog Astro —
+  Créez un référentiel GitHub pour votre projet de tutoriel
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Mettez votre projet en ligne
+</PreCheck>
+
+Ce tutoriel utilisera GitHub pour stocker notre dépôt et se connecter à un hébergeur web. Vous êtes libre d'utiliser le fournisseur git en ligne de votre choix.
+
+:::note
+Si vous êtes déjà familier avec git et avez votre propre flux de travail, créez un nouveau référentiel GitHub pour votre projet en utilisant votre méthode préférée. Passez à la page suivante : ["Déployez votre site sur le Web"](/en/tutorial/1-setup/5/).
```

Comment:
Always put `/fr/` on internals links, same if the URL doesn't exist (for the moment)